roxie/roxiemem/roxiemem.hpp

```diff
@@ -411,6 +419,7 @@ enum RoxieHeapFlags
     RHFvariable         = 0x0010,  // only used for tracing
     RHFblocked          = 0x0040,  // allocate blocks of rows
     RHFnofragment       = 0x0080,  // the allocated records will not be fragmented
+    RHFdelayrelease     = 0x0100,
```

Comment:
Should we add 0x100 to hintHeapFlags options for some Regression/PerformanceTesting ecl jobs ?